src/ecrc-frontend/src/components/page/transition/Transition.test.js

```diff
@@ -55,7 +55,7 @@ describe("Transition Page Component", () => {
       () => {
         expect(window.open).toHaveBeenCalled();
       },
-      { timeout: 6000 }
+      { timeout: 7000 }
```

Comment:
question: why does this need to be changed? the timeout in the code is still 6000 right?

Reply:
I just want to make sure it has time, the transition should only take 6000 but I don't know if Jenkins running slow would affect it so I figured it couldn't hurt.